inference-engine/src/transformations/CMakeLists.txt

```diff
@@ -23,15 +23,12 @@ source_group("include" FILES ${PUBLIC_HEADERS})
 
 add_library(${TARGET_NAME} SHARED ${LIBRARY_SRC} ${PUBLIC_HEADERS})
 
-target_compile_definitions(${TARGET_NAME} PRIVATE IMPLEMENT_INFERENCE_ENGINE_API)
+target_compile_definitions(${TARGET_NAME} PRIVATE TRANSFORMATIONS_DLL_EXPORTS)
```

Comment:
we can reuse `inference_engine_transformations_EXPORTS` which is defined automatically by cmake for dynamic libraries.

Reply:
Ok, will try to change